Frameworks/SafariServices/SFSafariWebViewController.mm

```diff
@@ -177,4 +207,8 @@ - (void)_cancel {
     [self dismissViewControllerAnimated:NO completion:nil];
 }
 
+- (void)dealloc {
+    _webView.delegate = nil;
```

Comment:
[UIWebView issue addressed in second commit.]

---
In reply to: [83342864](https://github.com/Microsoft/WinObjC/pull/1147#discussion_r83342864) [](ancestors = 83342864,83337136)